src/main/java/frc/robot/subsystems/climber/ClimberIOTalonFX.java

```diff
@@ -0,0 +1,304 @@
+  private StatusSignal<Double> rightSupplyCurrentAmpsStatusSignal;
+
+  private Alert configAlert =
+      new Alert("Failed to apply configuration for Climber.", AlertType.ERROR);
+
+  // FIXME: Add KV, KA, 
+  private final TunableNumber leftMotorKP =
+      new TunableNumber("Climber/leftMotorKP", LEFT_POSITION_PID_P);
+  private final TunableNumber leftMotorKI =
+      new TunableNumber("Climber/leftMotorKI", LEFT_POSITION_PID_I);
+  private final TunableNumber leftMotorKD =
+      new TunableNumber("Climber/leftMotorKD", LEFT_POSITION_PID_D);
+  private final TunableNumber leftMotorKS =
+      new TunableNumber("Climber/leftMotorKS", LEFT_POSITION_PID_KS);
+  private final TunableNumber leftMotorKV =
+      new TunableNumber("Climber/leftMotorKV", LEFT_POSITION_PID_KV);
+  private final TunableNumber leftMotorKA = 
+      new TunableNumber("Climber/leftMotorKA", LEFT_POSITION_PID_KA);
+  private final TunableNumber leftMotorKVExpo = 
+      new TunableNumber("Climber/leftMotorKVExpo", LEFT_POSITION_PID_KV_EXPO);
+  private final TunableNumber leftMotorKAExpo = 
+      new TunableNumber("Climber/leftMotorKAExpo", LEFT_POSITION_PID_KA_EXPO);
+  
+
+  private final TunableNumber rightMotorKP =
+      new TunableNumber("Climber/rightMotorKP", RIGHT_POSITION_PID_P);
+  private final TunableNumber rightMotorKI =
+      new TunableNumber("Climber/rightMotorKI", RIGHT_POSITION_PID_I);
+  private final TunableNumber rightMotorKD =
+      new TunableNumber("Climber/rightMotorKD", RIGHT_POSITION_PID_D);
+  private final TunableNumber rightMotorKS =
+      new TunableNumber("Climber/rightMotorKS", RIGHT_POSITION_PID_KS);
+  private final TunableNumber rightMotorKV = 
+      new TunableNumber("Climber/rightMotorKV", RIGHT_POSITION_PID_KV);
+  private final TunableNumber rightMotorKA = 
+      new TunableNumber("Climber/rightMotorKA", RIGHT_POSITION_PID_KA);
+  private final TunableNumber rightMotorKVExpo = 
+      new TunableNumber("Climber/rightMotorKVExpo", RIGHT_POSITION_PID_KV_EXPO);
+  private final TunableNumber rightMotorKAExpo = 
+      new TunableNumber("Climber/rightMotorKAExpo", RIGHT_POSITION_PID_KA_EXPO);
+
+  /** Create a TalonFX-specific generic SubsystemIO */
+  public ClimberIOTalonFX() {
+    leftVelocityRPSStatusSignal = leftMotor.getVelocity();
+    leftPositionMeterStatusSignal = leftMotor.getPosition();
+    leftStatorCurrentAmpsStatusSignal = leftMotor.getStatorCurrent();
+    leftSetpointStatusSignal =
+        leftMotor.getClosedLoopReference(); 
+
+    rightVelocityRPSStatusSignal = rightMotor.getVelocity();
+    rightPositionMetersStatusSignal = rightMotor.getPosition();
+    rightStatorCurrentAmpsStatusSignal = rightMotor.getStatorCurrent();
+    rightSetpointStatusSignal = rightMotor.getClosedLoopReference();
+
+    leftPositionCurrentRequest = new MotionMagicExpoTorqueCurrentFOC(0.0);
+    rightPositionCurrentRequest = new MotionMagicExpoTorqueCurrentFOC(0.0);
+
+    leftConfigMotor(LEFT_MOTOR_CAN_ID);
```

Comment:
use the same method for configuring left and right motors (pass the TalonFX reference as a parameter)